.changeset/strange-rice-juggle.md

```diff
@@ -0,0 +1,5 @@
+---
+"@frontify/frontify-cli": patch
+---
+
+fix: update template dependencies
```

Comment:
```suggestion
chore: bump dependency `@frontify/frontify-cli` to `^5.6.19` for the templates
```

Wdyt?

Reply:
sure thing, i still have some thing to update, ill add them then 👍

Reply:
🏁